infra/periodic_killer.tf

```diff
@@ -55,23 +55,34 @@ set -euxo pipefail
 apt-get update
 apt-get install -y jq
 
+echo "$(date -Is -u) boot" > /root/log
+
 cat <<CRON > /root/periodic-kill.sh
 #!/usr/bin/env bash
 set -euo pipefail
+echo "\$(date -Is -u) start"
```

Comment:
Why the escape here?

Reply:
This is creating the cron file (`cat <<CRON >> ...` on liine 60). We escape because we want the `date` command to run as part of the cron, not as part of creating the cron file (which would hardcode the machine creation time in the `periodic-kill.sh` file).

Reply:
Oh I missed that this was in the context of the `cat`, thanks!